templates/quiz/start-button.php

```diff
@@ -53,19 +56,54 @@
 						 * @param LLMS_Quiz   $quiz        The current quiz instance.
 						 * @param LLMS_Lesson $lesson      The parent lesson instance.
 						 */
-						echo apply_filters( 'lifterlms_begin_quiz_button_text', __( 'Start Quiz', 'lifterlms' ), $quiz, $lesson );
+						echo apply_filters( 'lifterlms_begin_quiz_button_text', $start_button_string, $quiz, $lesson );
```

Comment:
I'd escape the html on the result of this filter.

Reply:
escaped.